snippets/go/src/nb_jobs/README.md

```diff
@@ -0,0 +1,17 @@
+Call Netbackup APIs
+
+Calls the API and returns the data
+
+Call Get Admin/NetBackup/Jobs API and returns all Netbackup Admin Jobs
+
+Also, returns all the NetBackup jobs of the jobType 'Backup'
+
+
+#######Use following command to execute the program
```

Comment:
Please use markdown to format the readme. For example, `#` designates headings and indentation or backticks designate code.